xbmc/addons/AddonManager.cpp

```diff
@@ -111,6 +112,8 @@ bool CAddonMgr::Init()
     return false;
   }
 
+  m_trustedAddonRepos = StringUtils::Split(CCompileInfo::GetTrustedAddonRepos(), ',');
```

Comment:
I wonder if this belongs in the addonManager as a member variable or is better suited as part of AddonSystemSettings. I known it is not a standard setting but ends up being something similar to a one-time (compile time) definition injected from the build system.

Reply:
Looking at the signatures in AddonSystemSettings, I think it makes more sense as you have now, in the manager.